src/proxy/requirements.txt

```diff
@@ -7,3 +7,11 @@ cachetools>=5.3.2, <5.4.0
 asyncpg>=0.29.0, <0.30.0
 Black
 ruff
+azure-identity>=1.16.0, <1.17.0
+opentelemetry-api
+opentelemetry-sdk
+azure-monitor-opentelemetry-exporter
+opentelemetry-instrumentation-fastapi
+opencensus-ext-azure
+
```

Comment:
Random new lines. Also notice youre not doing version pinning, which gives me the heebie jeebies, but maybe you have a strategy for checking new versions before deployment.